src/events/http/HttpServer.js

```diff
@@ -412,6 +412,35 @@ export default class HttpServer {
     return authStrategyName
   }
 
+  _setAuthorizationStrategy(endpoint, functionKey, method, path) {
```

Comment:
Let's not prefix method names

Reply:
So I understand the desired change, do you not want the method prefixed with '_', 'set', or both?

Reply:
I meant `_` prefix. I don't think it's a convention here to prefix methods like that

Reply:
I noticed the rest of the file uses this convention so I was following the lead, happy to make the change, just want to ensure that it is still desired?

some examples from HttpServer.js

-  _extractAuthFunctionName(endpoint)
-  _configureAuthorization(endpoint, functionKey, method, path) 
-  _configureJWTAuthorization(endpoint, functionKey, method, path)
-  _extractJWTAuthSettings(endpoint)

Reply:
@tom-stclair indeed, I missed that. In such case feel free to leave it prefixed (still this convention remains to me unclear)